ehitchhiking-ui/src/app/shared/components/pre-loading/pre-loading.component.ts

```diff
@@ -0,0 +1,20 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {loaderSize} from '../../enums/pre-loader-sizes';
```

Comment:
The identifier of an enum should starts from capitalized letter

Reply:
fixed